release-team/shadows.md

```diff
@@ -0,0 +1,92 @@
+# Release Team Shadows
+
+The Release Team recruits a number of apprentices for each release role, in order to train new release team leads, handle tasks that each lead may not be able to cover, share knowledge about the release process, and help contributors broaden their areas of knowledge and participation.  These apprentices are called "Shadows" on the release team, and are equivalent to similar trainee positions on other teams, such as the Release Management Assistants.
+
+The Shadow system is an apprenticeship model, like those used by union tradespeople.  The idea is that Shadows learn by doing, while in the process taking load off the leads and improving the release.  After unplanned developments, they may even end up taking over for their lead.
+
+The following guide is for both Release role leads and shadows so that everyone knows what shadows are expected to do and when they are expected to do it.  In cases where the guide says "you," it is addressing the Shadow or potential Shadow.
+
+## Asking to be a Shadow
+
+In the last week of the prior release cycle, or the first week of the new one, the Emeritus Advisor and Release Lead will post a sign-up form for people who want to be Shadows.  Potential Shadows should fill out this form, which has a number of questions about their interests, availability, and experience; this is to judge who will be the most suitable and balance the team.  
+
+Please read the [Role Handbooks](https://github.com/kubernetes/sig-release/tree/master/release-team) for those roles before filling out the form.  Some handbooks will have a specific Shadow section that addresses what background Shadows are expected to have and what tasks they do.
+
+Shadow is an apprenticeship position, so candidates are not expected to be that experienced in contributing to Kubernetes or working on releases.
+
+During several release cycles, there have been three or more applicants for each Shadow spot, so many qualified candidates were turned down and asked to re-apply to a later team.  Role Leads, the EA and the Release Lead will choose Shadows based on a combination of suitability, availability, skills coverage, diversity, and compatible or covering time zones.
+
+## Week 1
+
+The first week should be used to get Shadows set up with all of the "paperwork" they need in order to participate fully in the Release Team:
+
+* Sign the [CNCF CLA](https://github.com/kubernetes/community/blob/master/CLA.md), which they need for all contributions.
+* Apply to be an Org Member if they are not already (see below).
+* Join the [#sig-release slack channel](https://slack.k8s.io/).
+* Get added to the `kubernetes-release-team` group by the Release Lead.
+* Join the [sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release) and [release-team](https://groups.google.com/forum/#!forum/kubernetes-release-team) Google Groups and calendars.
+* All all contact information to their Release Team's contact sheet.
```

Comment:
I think either there is a duplicate "all" or one "all was perhaps meant to be an "add"?